cts.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/jsonpath-standard/jsonpath-compliance-test-suite/main/cts.schema.json",
+  "title": "Compliance Test Suite",
+  "description": "JSONPath Compliance Test Suite",
+  "type": "object",
+  "properties": {
+    "tests": {
+      "description": "Individual test cases",
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/test_case"
+      }
+    }
+  },
+  "required": [
+    "tests"
+  ],
+  "$defs": {
```

Comment:
I think it's fine for all of these to just be inlined.  We're not really gaining anything from having these references.  If anything it hurts readability because we're having to read a def name then scan for what it means.

`$defs` is really good for deduplication, but since we're not defining the same schema for multiple places in the data, it doesn't matter.